src/main/java/io/mosip/mimoto/dto/mimoto/CredentialIssuerWellKnownResponse.java

```diff
@@ -0,0 +1,12 @@
+package io.mosip.mimoto.dto.mimoto;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class CredentialIssuerWellKnownResponse {
+    private String credential_issuer;
```

Comment:
All variables name should be in camelcase

Reply:
can we add javax.validations for this DTO / pojo , and run some javax validation to this serialized object, so that any change in the wellknown spec, is captured before the business logic.

Reply:
changed this